lib/container.ts

```diff
@@ -117,6 +118,13 @@ export class Container extends (EventEmitter as {
 
 			await this.addFiles(toAdd);
 			await this.deleteFiles(toDelete);
+
+			// Now we need to execute the commands
+			await Promise.all(
```

Comment:
I don't have the whole context but shouldn't the commands be executed in order?

Reply:
good catch!